opencti-platform/opencti-graphql/src/http/httpTaxii.js

```diff
@@ -138,6 +138,7 @@ const initTaxiiApi = (app) => {
       const stix = await restCollectionStix(context, user, collection, req.query);
       res.set('X-TAXII-Date-Added-First', getUpdatedAt(R.head(stix.objects)));
       res.set('X-TAXII-Date-Added-Last', getUpdatedAt(R.last(stix.objects)));
+      res.set('Content-Type', TAXII_VERSION);
```

Comment:
After looking a bit more, looks like we already add that part in a more generic way.

const extractUserFromRequest = async (context, req, res) => {
  res.setHeader('content-type', TAXII_VERSION);

So this line seems useless, can you check?

Reply:
Actually, this will work for `~/taxii2/root/collections/:id/objects` route used only. 

After some discussion, we want to apply in a more generic way on all routes